sdk/digitaltwins/dtdl-parser/src/parser/modelParserImpl.ts

```diff
@@ -205,7 +206,9 @@ export class ModelParserImpl implements ModelParser {
         );
       }
 
-      const additionalJsonTexts = await this.dtmiResolver(undefinedIdentifiers);
+      const additionalJsonTexts = await this.dtmiResolver(undefinedIdentifiers).then((value: {[dtmi: string]: unknown}) => {
```

Comment:
This seems like a change that needs to be taken to the parser node repo.

Reply:
yes but we will integrate it into the monorepo to make the changes more permanent and easier to manage.